katas/es6/language/generator/iteration.js

```diff
@@ -0,0 +1,43 @@
+// 50: Generators return iterable objects
```

Comment:
The top line should be
`// 50: Generator - iterator`
at some point I would like it to be generated from the __all__.js file.